polkadot-parachains/westmint/src/lib.rs

```diff
@@ -434,14 +434,15 @@ parameter_types! {
 	pub AssetsPalletLocation: MultiLocation =
 		PalletInstance(<Assets as PalletInfoAccess>::index() as u8).into();
 	pub CheckingAccount: AccountId = PolkadotXcm::check_account();
+	pub ParentAccount: AccountId = PalletId(*b"westend/").into_account();
```

Comment:
and this one has no prefix?

I think you should be following a standard here

Reply:
I need to have exactly 8 characters in `PalletId`, otherwise compilation fails and complains, so I have to make up some prefixes and suffixes.

Is there another ID type that I can use where I don't have the strict 8-character limit?

Reply:
look at other uses of `PalletId`. We prefix anything with parity as `py/` and then use some unique suffix.

`py` stands for parity, whereas probably some of these networks we should not directly connect with Parity since they are not owned or controlled by Parity.

As mentioned in the other chat, probably this is important enough to engage a consensus, and not just willy-nilly fill in something here that compiles.

This will be a permanent part of the network after you set it.

Reply:
Even using `PalletId` here might not be the right thing to do. Maybe you should even create your own struct which is `ParachainId` and allow more bytes to be used. All of these questions are not exactly clear, and so you need to engage the rest of the team.

Reply:
I've asked Gav before and he said that the hash of anything sensible is good enough. I think what would be good is to not use `PalletId` and instead use something that's more permissive here, and put in the name of the parent network to indicate the parent account, e.g. on Westmint, we'll have `Id("Westend")` and on Statemine, we'd have `Id("Kusama")`.